ee/ucp/swarm/deploy-to-collection.md

```diff
@@ -57,20 +57,30 @@ services:
 
   wordpress:
     image: wordpress
+    networks:
+      - wp
     ports:
       - 8080:80
     environment:
       WORDPRESS_DB_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
   mysql:
     image: mysql:5.7
+    networks:
+      - wp
     environment:
       MYSQL_ROOT_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
+
+networks:
+  wp:
```

Comment:
Alternatively, you can name this one `default:`, in which case it configures the default network for the stack (in that case it's not needed to reference the `wp` network in all services

Reply:
That's good to know! TIL.  I think I'll leave it for now as I think being more explicit in the documentation is better.